mitmproxy/addons/view.py

```diff
@@ -167,7 +167,7 @@ def __init__(self):
 
     def load(self, loader):
         loader.add_option(
-            "view_filter", typing.Optional[str], None,
+            "view_filter", typing.Optional[str], "",
```

Comment:
Is there a reason why you changed the default?

Reply:
There was a bug where if I tried to toggle the filter on it would return an attribute error none type object. Hence why I changed it.
![Screenshot_2021-08-20_00-08-40](https://user-images.githubusercontent.com/70739609/130083317-ed85fd96-0450-45ce-b942-854d827d01fb.png)